app/package-lock.json

```diff
@@ -0,0 +1,10062 @@
+{
```

Comment:
Uncommit the changes to app/package-lock.json because they're not related to the issue you're trying to solve.

I think the best way you could do this is by running commands like:
git reset HEAD~1
git add resources/assets/sass/_home.scss
git add resources/views/pages/home.blade.php
git commit -m "app: split Sign in and Sign up into separate links"